### PR TITLE
[2/2] Add EDS icons that require renaming existing files

### PR DESCRIPTION
**Pull request by anniehu4**
### Summary:
[ch191086]

Moves over files that already had a different icon name in the `next` branch. In almost every case, we use the EDS name instead since we decided name changing is outside the scope of this refactor (exceptions called out in inline comments). We'll likely be changing names once Alicia/Ryan are done with their audit instead.

Alas the files changed enough that Github doesn't show it as a rename 🥲 Specific mapping, format is `new-name (old-name)`:
- add (plus)
- autorenew (loop2)
- cancel (x-circle)
- close (x)
- create (pencil)
- delete (trash)
- error (exclamation-circle)
- expand-more (chevron-down)
- expand-less (chevron-up)
- help (question-mark-circle)
- info (information-circle)
- notifications (bell)
- open-in-new (external-link)
- remove (minus)
- schedule (clock)
- settings (cog)
- unfold-more (selector)
- warning (exclamation)

### Test Plan:
- [IconGrid in Storybook](https://www.chromatic.com/test?appId=61313967cde49b003ae2a860&id=62476939bff681003a4a54ad)

### CODE REVIEW

File: src/icons/copy.svg [hunk 1]
```diff
@@ -1,3 +1,3 @@
...
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
```
Comment (anniehu4) [inline src/icons/copy.svg:1]: The EDS name is `"file-copy"`, but `"copy"` is shorter and sweeter

File: src/icons/dots-vertical.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
```
Comment (anniehu4) [inline src/icons/dots-vertical.svg:1]: this name in EDS is `"more-vert"` which I really hate 😂  there's also already an icon in this branch named `dots-horizontal` so this is a nice parallel

Comment (dierat) [reply src/icons/dots-vertical.svg:1]: OR we rename `dots-horizontal` to `more-hor`

File: src/components/Icon/Icon.stories.tsx [hunk 3]
```diff
@@ -15,6 +15,6 @@ Default.args = { name: 'x' };
 
 export const Inverted = () => (
   <div className="u-padding-sm" style={{ background: '#000000' }}>
-    <Icon inverted={true} name="x" />
```
Comment (dierat) [inline src/components/Icon/Icon.stories.tsx:18]: Yaaay I dunno why but this one really bugged me.

**Comment by anniehu4** [comment]
oop I still need to update some of the story icons

**Review by dierat — APPROVED** [review]
Awesome!

**Pull request merged**